conftest.py

```
@@ -898,6 +898,52 @@ def prepared_plan(
+        prepared_plan (dict[str, Any]): Base prepared plan with cloned VMs
+
+    Yields:
+        dict[str, Any]: Deep copy of prepared plan with first VM only
+    """
+    plan: dict[str, Any] = deepcopy(prepared_plan)
+    # Use only the first VM for plan 1
+    plan["virtual_machines"] = [plan["virtual_machines"][0]]
+    # Copy source VM data for the first VM only
+    vm_name = plan["virtual_machines"][0]["name"]
+    plan["source_vms_data"] = {vm_name: prepared_plan["source_vms_data"][vm_name]}
+    yield plan
```

Ruff (0.14.14)

[warning] No teardown in fixture `prepared_plan_1`, use `return` instead of `yield`

Replace `yield` with `return`

(PT022)